dockerhub/travis-ci-image.dockerfile

```diff
@@ -51,6 +51,7 @@ RUN locale-gen $LANG
 
 #NOTE(sileht): Upgrade python dev tools
 RUN pip install -U pip tox virtualenv
+RUN pip install zipp configparser
```

Comment:
what's this for?

Reply:
Fix some broken dependency stuff when trying to build the new 16.04 image, didn't bother checking it since the plan is to move away from 16.04 to a newer version just need a new image to see if it solved gnocchi CI completely.